pkg/trustedresources/verify.go

```diff
@@ -157,13 +211,47 @@ func prepareObjectMeta(in metav1.ObjectMeta) (metav1.ObjectMeta, []byte, error)
 	return out, signature, nil
 }
 
-// getVerifiers get all verifiers from configmap
-func getVerifiers(ctx context.Context, k8s kubernetes.Interface) ([]signature.Verifier, error) {
-	cfg := config.FromContextOrDefaults(ctx)
+// getVerifiersFromPolicy get all verifiers from configmap
+func getVerifiersFromPolicy(ctx context.Context, k8s kubernetes.Interface, source string, verificationpolicy *v1alpha1.VerificationPolicy) ([]signature.Verifier, error) {
 	verifiers := []signature.Verifier{}
```

Comment:
nit: Might be useful to pass back even the authority name along with the actual verifiers, so that we can log (or even generate provenance) on what authority and pattern were used to verify this resource.

Reply:
sg, mark a todo for this!